espnet2/enh/espnet_model.py

```diff
@@ -38,6 +41,10 @@ def __init__(
         mask_type: Optional[str] = None,
         flexible_numspk: bool = False,
         extract_feats_in_collect_stats: bool = False,
+        normalize_variance: bool = False,
+        normalize_variance_per_ch: bool = False,
+        categories: list = [],
+        category_weights: list = [],
     ):
```

Comment:
I feel like we need to add a little more doc string here, given that we have quite a few parameters passed in this init function.

Reply:
done